frontend/src/components/SimpleGetRequestTemplate.tsx

```diff
@@ -1,16 +1,43 @@
 import React, { useState } from 'react';
 import axios from 'axios';
```

Comment:
Remove this library if it's not used to fix lint

Reply:
fixed just now